src/Glpi/Http/SymfonyRequest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2025 Teclib' and contributors.
+ * @copyright 2003-2014 by the INDEPNET Development Team.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Http;
+
+use Symfony\Component\HttpFoundation\Request;
+
+use function Safe\preg_replace;
+
+class SymfonyRequest extends Request
+{
+    public function __construct(array $query = [], array $request = [], array $attributes = [], array $cookies = [], array $files = [], array $server = [], $content = null)
+    {
+        // Normalize plugins paths.
+        // All plugins resources should now be accessed using the `/plugins/${plugin_key}/${resource_path}`.
+        $pattern = '#^/marketplace/#';
```

Comment:
This will not work if GLPI URL is prefixed (e.g. `http://domain.tld/glpi/marketplace/glpiinventory/`).

I wonder if this alternative patch would fix the issue:
```diff
diff --git a/src/Plugin.php b/src/Plugin.php
index 5a5f61385e..3276cfd465 100644
--- a/src/Plugin.php
+++ b/src/Plugin.php
@@ -133,7 +133,7 @@ class Plugin extends CommonDBTM
      *      `plugin_key`: the plugin key;
      *      `plugin_resource`: the resource path relative to the plugin.
      */
-    public const PLUGIN_RESOURCE_PATTERN = '#^/plugins/(?<plugin_key>[^/]+)(?<plugin_resource>/.*)$#';
+    public const PLUGIN_RESOURCE_PATTERN = '#^/(?:plugins|marketplace)/(?<plugin_key>[^/]+)(?<plugin_resource>/.*)$#';
 
     /**
      * Plugin key validation pattern.
```

Reply:
As said in our internal channel, I already tried to change the constant.

I'll rework to support prefixed URI